cmd/stack_package.go

```diff
@@ -88,9 +91,43 @@ func newStackPackageCmd(rootConfig *RootCommandConfig) *cobra.Command {
 			Info.Log("Running appsody stack package")
 			Info.Log("******************************************")
 
-			stackPath := rootConfig.ProjectDir
+			projectPath := rootConfig.ProjectDir
+
+			// sets stack path to be the copied folder
+			stackPath := projectPath + "stackPath"
```

Comment:
This is a temporary directory to apply the templating and run the package. I don't think we should be creating it in the user's projectPath. First, it could conflict with user's real files. Second, if something goes wrong and files are left there, the user may not like this and it may show up in git if they track this dir.

A better location would either be in `/tmp` or `~/.appsody`. We've used the appsody dir for other temp artifacts like for extract and logs, so I think this makes most sense. Here's what I'm thinking:
`stackPath := filepath.Join(getHome(rootConfig), "stacks", "packaging-" + stackID)`

Reply:
Agreed, this is a much better approach, @neeraj-laad and I discussed doing this last Friday, I'll implement this today